NOMENCLATURE.md

```diff
@@ -0,0 +1,56 @@
+# Glossary and Nomenclature - General
+
+## Aztec Labs
+The company - aztec.network
+
+## Aztec Protocol
+Definition of a layer 2 network that rolls up to Ethereum. Includes how to execute public/private functions, verify proofs, and reaches consensus on shared state.
+
+## Aztec Network
+Computers on a network that adhere to the Aztec Protocol.
+
+## Aztec Client Software
+Software run on individual computers that adhere to the Aztec protocol.
+(learn more from the Yellow Paper)
+
+## Aztec (Public) Virtual Machine
+Executes the public section of a transaction, and unconstrained functions.
+
+## Private eXecution Environment (PXE)
+Execute the private section of a transaction.
+
+## Noir Language
+A rust-like language to represent constraints for zero-knowledge circuits.
```

Comment:
"Rust-like" can be confusing. it has syntax familiar to rust, but is very different. 

Consider the tagline from the Noir website

Noir is a Domain-Specific Language for SNARK proving systems developed by [Aztec Labs](https://aztec.network/). It allows you to generate complex Zero-Knowledge Programs (ZKP) by using simple and flexible syntax, requiring no previous knowledge on the underlying mathematics or cryptography.

Reply:
Done.
Interestingly aztec.nr says: "Rust-like syntax"

Reply:
It does have Rust-like syntax, but it's not a Rust-like language